scripts/format

```diff
@@ -15,3 +15,5 @@ find oak examples toolchain \( -type f -name BUILD -o -name '*bzl' \) -exec buil
 find oak examples \( -type f -name '*.h' -o -name '*.cc' -o -name '*.proto' \) \
     -exec clang-format -i -style=file {} +
 find examples rust -type f -name '*.rs' -exec rustfmt {} +
+
+prettier --write ./*.md ./docs/*.md
```

Comment:
Does the initial `mdformat` need to get dropped too?

Reply:
Done.